### PR TITLE
Homework6: Alexey Kozhan

### PR DESCRIPTION
**Pull request by AlexKozhan**
Homework6 is done in main.py file

### CODE REVIEW

File: homework.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+"""Докстрингс"""
```
Comment (Stml89) [inline homework.py:1]: Are you joking and please use English language

File: homework.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+
+# Перевести строку в список "Robin Singh" => ["Robin”, “Singh"]
+FIRST_LINE = "Robin Singh"
+print(list(FIRST_LINE.split(" ")))
```
Comment (Stml89) [inline homework.py:6]: `white space` by default: FIRST_LINE.split()

File: homework.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+
+# "I love arrays they are my favorite" =>
+# ["I", "love", "arrays", "they", "are", "my", "favorite"]
+SECOND_LINE = "I love arrays they are my favorite"
+print(list(SECOND_LINE.split(" ")))
```
Comment (Stml89) [inline homework.py:12]: same

File: homework.py (hunk 1 above)
Comment (AlexKozhan) [reply homework.py]: i dont't understand what i need to put in dockstrings in this homework. There are no function, class, or any complex method definition.

File: homework.py (hunk 2 above)
Comment (AlexKozhan) [reply homework.py]: done

File: homework.py (hunk 3 above)
Comment (AlexKozhan) [reply homework.py]: done

File: homework.py (hunk 1 above)
Comment (AlexKozhan) [reply homework.py]: perhaps i've succeeded

**Comment by AlexKozhan** [comment]
dockstrings fixed

**Review by Stml89 — APPROVED** [review]

**Pull request merged**